### PR TITLE
mount cache folder when building wheels

### PR DESCRIPTION
**Pull request by siyuan0322**
## Related issue number

Fixes

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -147,15 +145,59 @@ jobs:
...
-        sudo make graphscope-py3-package
+        mkdir -p ${HOME}/.m2 ${HOME}/.cache/sccache ${HOME}/.cargo/git ${HOME}/.cargo/registry
+        docker pull registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-vineyard:v0.3.11
+        docker run --rm \
+          -v ${GITHUB_WORKSPACE}:/work \
+          -v ${HOME}/.m2:/home/graphscope/.m2 \
+          -v ${HOME}/.cache/sccache:/home/graphscope/.cache/sccache \
+          -v ${HOME}/.cargo/git:/home/graphscope/.cargo/git \
+          -v ${HOME}/.cargo/registry:/home/graphscope/.cargo/registry \
+          registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-vineyard:v0.3.11 \
```
Comment (sighingnow) [inline .github/workflows/ci.yml:159]: Move the mount options to the makefile. You could add an option to disable it, but please enable cache by default. You have have something like

```
NO_CACHE := false

ifeq ($(NO_CACHE), true)
    CACHE_MOUNT_OPTS =
else
    CACHE_MOUNT_OPTS = -v $.....
endif

graphscope-manylinux2014-py3:
    ....
    docker run --rm $CACHE_MOUNT_OPTS ....
```

**Review by sighingnow — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sighingnow) [reply .github/workflows/ci.yml:159]: sccache could be installed to our graphscope-runtime image. As it is only used for building wheels.

Comment (siyuan0322) [reply .github/workflows/ci.yml:159]: Let's see if this can work. Then I would move it to makefile.

Comment (lidongze0629) [reply .github/workflows/ci.yml]: > 

+1

**Review by lidongze0629 — APPROVED** [review]

**Review by sighingnow — APPROVED** [review]
LGTM.

**Pull request merged**